Source/Model/Message/ZMOTRMessage.h

```diff
@@ -21,13 +21,15 @@
 
 @class UserClient;
 @class MessageUpdateResult;
+@class ButtonState;
 
 NS_ASSUME_NONNULL_BEGIN
 
 extern NSString * const DeliveredKey;
 
 @interface ZMOTRMessage : ZMMessage
 
+@property (nullable, nonatomic, retain) NSSet<ButtonState *> *buttonStates;
```

Comment:
```suggestion
@property (nonatomic, nullable) NSSet<ButtonState *> *buttonStates;
```

- Fix order of annotations
- `retain` is the default and therefore be left out.